src/include/autodetectcompilers.h

```diff
@@ -13,11 +13,41 @@ class wxUpdateUIEvent;
 
 class AutoDetectCompilers : public wxScrollingDialog
 {
+    private:
+        enum CompilerHighlightColor
+        {
+            eHighlightNone = 0,
+            eHighlightGrey,
+            eHighlightGreen,
+            eHighlightRed,
+            eHighlightYellow
+        };
+
+        enum CompilerColumnsNumber
+        {
+            ccnNameColumn = 0,
+            ccnStatusColumn,
+            ccnDetectedPathColumn,
+        };
+
+        struct CompilerItem
+        {
+            wxString wxsCompilerName;
+            wxString wxsStatus;
+            wxString wxsCompilerPath;
+            CompilerHighlightColor eHighlight;
+            bool bDetected;
+        };
+        std::vector<CompilerItem> vCompilerList;
```

Comment:
Indent here is a bit odd, please fix.

Reply:
vCompilerList doesn't follow the convention for members.
I prefer to put members at the bottom of the class.

Using a vector requires an include. This would fail no pch builds.

Reply:
Any updated on the indenting? Can you check your editor settings as it looks okay on my end.